r/tests/testthat/test-info.R

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Please rename this file to test-arrow-info.R, or, since you haven't actually added a test, move this back to where it was before.